infra/aws/terraform/policy-staging-account-1/terraform.tf

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+terraform {
+  backend "s3" {
+    bucket = "k8s-aws-root-account-terraform-state"
```

Comment:
why not use a bucket local to the policy staging account ? so we can deploy this yourself ? feel to create the bucket manually if needed. 

we want to test policies application.

Reply:
@ameukam I've changed the Terraform state bucket to use a bucket in the policy staging account as suggested.